integration_tests/pages/page.ts

```diff
@@ -40,4 +40,19 @@ export default abstract class Page {
   signOut = (): PageElement => cy.get('[data-qa=signOut]')
 
   manageDetails = (): PageElement => cy.get('[data-qa=manageDetails]')
+
+  hasNoErrors() {
```

Comment:
These methods are in the superclass `Page` (which all page classes inherit from), so we can put common methods like this in here 👍